src/scene-manager.ts

```diff
@@ -257,6 +281,9 @@ export class SceneManager {
 
     this.initScene();
     this.animate();
+    this.eventsDispatcher.addEventListener(SceneManagerEvent.ANIMATION_COMPLETE, () => {
+      this.setRerenderListeners();
+    });
```

Comment:
That's the key to re-render automatically on changes. This is setting listeners on all properties that we want to see reflected right away on the scene.